bitrise.yml

```diff
@@ -52,8 +53,8 @@ workflows:
               echo "BITRISE_TEST_DIR_PATH (\"$BITRISE_TEST_DIR_PATH\") should be: \"$BITRISE_DEPLOY_DIR/Debug-iphoneos\""
               exit 1
             fi
-            if [[ "$BITRISE_DEPLOY_DIR/ios-simple-objc_iphoneos12.0-arm64e.xctestrun" != "$BITRISE_XCTESTRUN_FILE_PATH" ]] ; then 
-              echo "BITRISE_XCTESTRUN_FILE_PATH (\"$BITRISE_XCTESTRUN_FILE_PATH\") should be: \"$BITRISE_DEPLOY_DIR/ios-simple-objc_iphoneos12.0-arm64e.xctestrun\""
+            if [[ "$BITRISE_DEPLOY_DIR/renamed_scheme_iphoneos12.1-arm64e.xctestrun" != "$BITRISE_XCTESTRUN_FILE_PATH" ]] ; then 
```

Comment:
use the scheme env (see my previous comment) for creating the file name in the path.